kubespawner/spawner.py

```diff
@@ -1313,6 +1347,11 @@ def get_pod_manifest(self):
         else:
             real_cmd = None
 
+        if self.namespace_name_template:
+            service_account = self.namespace
```

Comment:
Maybe the name shouldn't exactly *be* the namespace, but derived from it?

Reply:
In EG we use the pattern where we suffix the RBAC resources with something that indicates what it is, but in this case, I was following the same pattern as JupyterHub that creates hub as the name of the namespace, service account, role, role binding. Should I depart from the JupyterHub pattern ?